src/machinelearningservices/azext_mlv2/manual/requirements.txt

```diff
@@ -0,0 +1,5 @@
+cryptography
```

Comment:
Are these dependencies required by the ML extension? If so, please add them to the setup.py file so that they will be automatically installed when the extension is installed.